doc/dev/developer_guide/dash-devel.rst

```diff
@@ -1026,6 +1026,59 @@ Alerts and notifications
 Default notification should have `text-info` color. Success notification should
 have `text-success` color. Failure notification should have `text-danger` color.
 
+Accessibility
+.............
+
+We follow the `W3C WCAG2.1 <https://www.w3.org/TR/WCAG21/>`_  level A accessibility conformance guidelines. 
+By implementing accessibility best practices, you are also improving the usability of the site for all users.
+
+**Accessibility checkers:** During development, you can test the accessibility compliance of your features using one of the tools below
+
+- `Accessibility insights plugin <https://accessibilityinsights.io/downloads/>`_
+- `Site Improve plugin <https://www.siteimprove.com/integrations/browser-extensions/>`_
+- `Axe devtools <https://www.deque.com/axe/devtools/>`_
+
+Testing with two or more of these tools can greatly improve the detection of possible accessibility violations
+
+**Color contrast checkers:** When adding new colors, making sure they are accessible is also important, Here are some tools which can help with color contrast testing
+
+- `Accessible web color-contrast checker <https://accessibleweb.com/color-contrast-checker/>`_
+- `Colorsafe generator <https://colorsafe.co/>`_
+
+**Accessibility linters:** If you use vscode, we recommend installing the `axe accessibility linter <https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter>`_
+which can help you catch and fix potential issues during development
+
+**Accessibility testing:** Our e2e testing suite based on cypress supports the addition of accessibility tests using `axe-core <https://github.com/dequelabs/axe-core>`_ 
+and `cypress-axe <https://github.com/component-driven/cypress-axe>`_. A custom cypress command `cy.checkAccessibility` can also be used directly. 
+This is a great way to prevent regressions on high impact components.
```

Comment:
```suggestion
**Accessibility testing:** Our e2e testing suite, which is based on Cypress, supports the addition of accessibility tests using `axe-core <https://github.com/dequelabs/axe-core>`_ 
and `cypress-axe <https://github.com/component-driven/cypress-axe>`_. A custom Cypress command, `cy.checkAccessibility`, can also be used directly. 
This is a great way to prevent accessibility regressions on high impact components.
```